deployments/optimism-goerli/usdc/configuration.json

```diff
@@ -24,6 +24,14 @@
     "baseMinForRewards": "1000000e6"
   },
   "assets": {
+    "OP": {
```

Comment:
missing `address`?

Reply:
It doesn't end up in `configuration.json`. But it will be part of `aliases.json` when you spider.

Reply:
ah, that's right. `getContractAddress` pulls the contract by its alias